plugins/newsdownloader.koplugin/main.lua

```diff
@@ -0,0 +1,185 @@
+        f:close()
+        return deserializeXMLString(xmltext)
+    else
+        logger.warn("NewsDownloader: XML file not found", filename, e)
+    end
+end
+
+function NewsDownloader:init()
+    self.ui.menu:registerToMainMenu(self)
+end
+
+function NewsDownloader:addToMainMenu(menu_items)
+    if not initialized then
+        NEWS_DL_DIR = string.format("%s/%s/", DataStorage:getDataDir(), NEWS_DL_DIR_NAME)
+        if not lfs.attributes(NEWS_DL_DIR, "mode") then
+            lfs.mkdir(NEWS_DL_DIR)
+        end
+
+        FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
+        if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
+            logger.dbg("NewsDownloader: Creating init configuration")
+            FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
+                             FEED_CONFIG_PATH)
+        end
+        initialized = true
+    end
+
+    menu_items.rss_news_downloader = {
+        text = _("Simple News(RSS/Atom) Downloader"),
+        sub_item_table = {
+            {
+                text = _("Download news"),
+                callback = function() self:loadConfigAndProcessFeeds() end,
+            },
+            {
+                text = _("Go to news folder"),
+                callback = function()
+                    local FileManager = require("apps/filemanager/filemanager")
+                    if FileManager.instance then
+                        FileManager.instance:reinit(NEWS_DL_DIR)
+                    else
+                        FileManager:showFiles(NEWS_DL_DIR)
+                    end
+                end,
+            },
+            {
+                text = _("Remove news"),
+                callback = function()
+                    -- puerge all downloaded news files, but keep the feed config
+                    for entry in lfs.dir(NEWS_DL_DIR) do
+                        if entry ~= "." and entry ~= ".." and entry ~= FEED_CONFIG_FILE then
+                            local entry_path = NEWS_DL_DIR .. "/" .. entry
+                            local entry_mode = lfs.attributes(entry_path, "mode")
+                            if entry_mode == "file" then
+                                ffi.C.remove(entry_path)
+                            elseif entry_mode == "directory" then
+                                FFIUtil.purgeDir(entry_path)
+                            end
+                        end
+                    end
+                    UIManager:show(InfoMessage:new{
+                        text = _("All news removed.")
+                    })
+                end,
+            },
+            {
+                text = _("Help"),
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = T(_("Plugin reads feeds config file: %1, and downloads their news to: %2. News limit can be set. To set you own news sources edit feeds config file. Only RSS, Atom is currently not supported."),
```

Comment:
How about something like this?

"The newsfeed downloader can be configured in the feeds config file:\n%1\n\nIt downloads the news items to:\n%2.\n\nA new item limit can be set. To set you own news sources edit feeds config file.\n\nFor now the newsfeed downloader only supports RSS. Atom is currently not supported."